kolibri/core/theme_hook.py

```diff
@@ -39,6 +40,9 @@
 TITLE = "title"
 TITLE_STYLE = "titleStyle"
 TOP_LOGO = "topLogo"
+LOGO = "logo"
+BRANDED_FOOTER = "brandedFooter"
+TEXT_LINES = "textLines"
```

Comment:
I might name this `PARAGRAPH_ARRAY` or similar to be more explicit about the expect data type and output